Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BaseTeleOp.java

```diff
@@ -21,15 +23,16 @@ public void driveRobot() {
         double angle = Math.toDegrees(Math.atan2(y, x)); // 0 degrees is forward
         double power = calculateDistance(x, y);
 
-        driveMecanum(angle, power, rotationalPower);
+        //driveMecanum(angle, power, rotationalPower);
+        driveMecanumGyro(angle, power, rotationalPower);
     }
 
     public void driveRobotSpeed() {
         isSlowMode = driveSpeedToggle.toggle(gamepad1.left_bumper);
         if (isSlowMode) {
-            driveSpeed = 0.50;
+            driveSpeed = 0.3;
         } else {
-            driveSpeed = 1.0;
+            driveSpeed = 0.8;
```

Comment:
Technically it would be better to have different constant variables for slow drive speed and regular drive speed to avoid having magic numbers here. https://en.wikipedia.org/wiki/Magic_number_(programming)